src/rvoice/fluid_rev.c

```diff
@@ -481,11 +481,31 @@ fluid_revmodel_processmix(fluid_revmodel_t* rev, fluid_real_t *in,
 }
 
 static void
+/* scalelevelwidth is compensation weigth factor to get output
+   amplitude rather  independent of width setting.
+   0: the output amplitude is fully dependant of width setting.
+   >0: the output amplitude is less dependant of width setting.
+   With a scalelevelwidthof of 0.2 the output amplitude is 
+   rather independent of width setting.
+*/
+#define scalelevelwidth 0.2f 
```

Comment:
This is a pretty unsuitable location to place a define. Please place it some lines above (around 242) where the other `scale*` defines are located.